operator/controllers/manifest_controller_helper_test.go

```diff
@@ -118,32 +118,18 @@ func createImageSpec(name, repo string, ociLayerType OCILayerType) manifestTypes
 	return imageSpec
 }
 
-func NewTestManifest(name string, componentOwner string) *v1alpha1.Manifest {
+func NewTestManifest() *v1alpha1.Manifest {
 	return &v1alpha1.Manifest{
-		TypeMeta: metav1.TypeMeta{
-			Kind: "Manifest",
-		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      name + RandString(8),
+			Name:      string(uuid.NewUUID()),
```

Comment:
IMO, this uuid is not a good solution, with this randomized uuid, we don't know which manifest CR from log, if go this way, at least have a prefix which can define the name.

Reply:
I agree, let me add a prefix